packages/react-is/src/__tests__/ReactIs-test.js

```diff
@@ -131,9 +133,10 @@ describe('ReactIs', () => {
 
   it('should identify memo', () => {
     const Component = () => React.createElement('div');
-    const memoized = React.memo(Component);
-    expect(ReactIs.typeOf(memoized)).toBe(ReactIs.Memo);
-    expect(ReactIs.isMemo(memoized)).toBe(true);
+    const Memoized = React.memo(Component);
+    expect(ReactIs.typeOf(Memoized)).toBe(ReactIs.Memo);
+    expect(ReactIs.typeOf(<Memoized />)).toBe(ReactIs.Element);
```

Comment:
> I'm not sure if this is by design or intentional, but it does seem inconsistent with how isMemo works.

As I mentioned above,

```js
// why is this ReactIs.ForwardRef
expect(ReactIs.typeOf(<RefForwardingComponent />)).toBe(ReactIs.ForwardRef);

// but this is ReactIs.Element 😕 
expect(ReactIs.typeOf(<Memoized />)).toBe(ReactIs.Element);
```

Reply:
Resolved in #17278

See https://github.com/facebook/react/pull/17278#discussion_r342690034